media-libs/libsdl3/libsdl3-3.2.4.recipe

```diff
@@ -0,0 +1,74 @@
+SUMMARY="Simple Direct Media Layer 3.0"
+DESCRIPTION="Simple DirectMedia Layer is a cross-platform development library \
+designed to provide low level access to audio, keyboard, mouse, joystick, and \
+graphics hardware via OpenGL and Direct3D. It is used by video playback \
+software, emulators, and popular games."
+HOMEPAGE="https://www.libsdl.org/"
+COPYRIGHT="1997-2024 Sam Lantinga"
+LICENSE="Zlib"
+REVISION="1"
+SOURCE_URI="https://www.libsdl.org/release/SDL3-$portVersion.tar.gz"
+CHECKSUM_SHA256="2938328317301dfbe30176d79c251733aa5e7ec5c436c800b99ed4da7adcb0f0"
+SOURCE_DIR="SDL3-$portVersion"
+# PATCHES=""
+
+ARCHITECTURES="all"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+    libsdl3$secondaryArchSuffix = $portVersion compat >= 3.0
+    lib:libsdl3$secondaryArchSuffix = 0.200.4 compat >= 0
```

Comment:
Use libVersionCompat for this, also version is wrong here:
```
-- Installing: /packages/libsdl3-3.2.4-1/.self/lib/libSDL3.so.0.2.4
-- Installing: /packages/libsdl3-3.2.4-1/.self/lib/libSDL3.so.0
-- Installing: /packages/libsdl3-3.2.4-1/.self/lib/libSDL3.so
```

Reply:
I have changed the names to uppercase as instructed.
And regarding the other issue, I was not really sure about the versioning part so I just followed the sdl2 port
```
	libsdl2${secondaryArchSuffix}_devel = $portVersion compat >= 2.0
	cmd:sdl2_config$secondaryArchSuffix = $portVersion compat >= 2.0
	devel:libSDL2$secondaryArchSuffix = 0.3000.10 compat >= 0
	devel:libSDL2_2.0$secondaryArchSuffix = 0.3000.10 compat >= 0
	devel:libSDL2main$secondaryArchSuffix
```
It seems to use 0.3000.10 for v2.30.10. I tried checking the build recipe docs but it isn't very clear about this either. Can you kindly guide me here?

Reply:
Pushing rest of the changes.

Reply:
The output you pasted is the output created by the recipe, not what's actually in there, for libversion you can check the install log near the end of the build and see the output I posted earlier.
You can also double click the created package in `~/haikuports/packages` to see it's content.

Reply:
Uhm, what I pasted is not any output. It's taken from here: https://github.com/haikuports/haikuports/blob/095aa6c1270671880d1973e4e6931f016863d046/media-libs/libsdl2/libsdl2-2.30.10.recipe#L28

Reply:
I hope it's correct this time?